Tools/scripts/summarize_stats.py

```diff
@@ -179,6 +192,16 @@ def gather_stats(input):
                     value = int(value)
                     stats[key] += value
             stats['__nfiles__'] += 1
+
+        import opcode
```

Comment:
Just curious, why was this import moved here? Just to reduce the temptation to use it elsewhere in the file?

Reply:
Yes, exactly.  Any use of `opcode`, `dis` or other sorts of bytecode introspection things on the "compare two results" path could lead to a subtle, silent error.  I'll add a comment.